admin_guide/manage_rbac.adoc

```diff
@@ -574,7 +565,7 @@ Subjects:
 ----
 
 [[managing-role-bindings]]
-== Managing Role Bindings
+== Managing role bindings
```

Comment:
Did you mean to remove the [[viewing-local-bindings]] on line 523?

Reply:
Yes. The collection had one link to one example, and I think that a link to the nearest section head works as well.